### PR TITLE
LABS-448 align mongo repo branch and labs-modules branch

### PR DESCRIPTION
**Pull request by steve-1820**

### CODE REVIEW

File: jstests/stream/checkpoint.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Testing checkpoints
 
-load("src/mongo/db/modules/labs-modules/jstests/stream/utils/stream.js");
+load("src/mongo/db/modules/labs/jstests/stream/utils/stream.js
```
Comment (Z-Shang) [inline jstests/stream/checkpoint.js:3]: missing a `");` at the end of line?

File: jstests/stream/create_stream.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // Testing create stream
 
-load("src/mongo/db/modules/labs-modules/jstests/stream/utils/stream.js");
+load("src/mongo/db/modules/labs/jstests/stream/utils/stream.js
```
Comment (Z-Shang) [inline jstests/stream/create_stream.js:3]: missing a "); at the end of line?

**Review by Z-Shang — CHANGES_REQUESTED** [review]

File: jstests/stream/checkpoint.js (hunk 1 above)
Comment (steve-1820) [reply jstests/stream/checkpoint.js:3]: @Z-Shang great catch - will fix

**Review by Z-Shang — APPROVED** [review]

**Review by michaelcahill — APPROVED** [review]
lgtm

**Pull request merged**